integr8ly/integreatly-operator
# change icons in openshift ui

# Description
[MGDAPI-2478](https://issues.redhat.com/browse/MGDAPI-2478) Updating RHOAM icon to new image.


## Verification
Deploy Operator with icon changes to a cluster with OLM and bundle format.

1. Checkout MGDAPI-2478 branch
2. Ensure you have the following repositories in your private org :

- managed-api-service
- managed-api-service-bundle
-  managed-api-service-index

3. Generate new csv using prepare-release.sh script.` ORG=<quay_org> OLM_TYPE=managed-api-service SEMVER=1.12.0 ./scripts/prepare-release.sh`
4. Build  and push operator image using  `ORG=<quay_org> INSTALLATION_TYPE=managed-api make image/build`  and  `ORG=<quay_org> INSTALLATION_TYPE=managed-api make image/push` .Note make sure repo is tagged correctly  eg.`rhoam-v1.12.0`
6. log in to the cluster.
6. Run `bundle-rhmi-operator.sh` script to build and push bundles and indexes to repo
7. login to cluster and check that images have been updated as in the screenshot below
![Screenshot from 2021-08-16 17-25-47](https://user-images.githubusercontent.com/74991829/129597072-0a3fab65-2574-49d5-8240-6457f5d193af.png)


## Type of change




## Checklist
